protostar/configuration_file/configuration_file_v2.py

```diff
@@ -109,37 +111,49 @@ def read(
     ) -> ConfigurationFileV2Model:
         raise NotImplementedError("Operation not supported.")
 
-    def create_file_content(
+
+class ConfigurationFileV2ContentConfigurator(
```

Comment:
```suggestion
class ConfigurationFileV2ContentFactory(
```

Reply:
Done in [9f01b4c](https://github.com/software-mansion/protostar/pull/897/commits/9f01b4ce49e21154b6d553b17c2803d413761e9f)